legacy_packages/react/src/wallet/wallets/embeddedWallet/CountrySelector.tsx

```diff
@@ -0,0 +1,99 @@
+import { useQuery } from "@tanstack/react-query";
+import React, { useRef } from "react";
+import { fontSize, spacing } from "../../../design-system";
+import { useCustomTheme } from "../../../design-system/CustomThemeProvider";
+import { StyledSelect } from "../../../design-system/elements";
+
+export function CountrySelector({
+  countryCode,
+  setCountryCode,
+}: {
+  countryCode: string;
+  setCountryCode: React.Dispatch<React.SetStateAction<string>>;
+}) {
+  const selectRef = useRef<HTMLSelectElement>(null);
+
+  const { data: supportedCountries } = useQuery({
+    queryKey: ["supported-sms-countries"],
+    queryFn: async () => {
+      const { supportedSmsCountries } = await import(
+        "@thirdweb-dev/wallets/evm/wallets/embedded-wallet"
+      );
```

Comment:
this will import the whole file, I assume there is more than just the supportedSMSCountries inside that file? Can we point this at a more granular file?

Reply:
correct, ideally we can point to a specific file somehow, but I guess that might be hard to do in v4

Reply:
(because cross package etc)